### DISCUSSION TITLE
There is no permission named `1` for guard `web`.

### DISCUSSION THREAD

**Discussion by drbyte** [spatie/laravel-permission#2541] [Q&A]
@mohammedALhakmy wrote:

@nille  this code of error with use splite laravel permission , laravel 10 and site the three class in the 
add roles insert show the error why 

There is no permission named `1` for guard `web`.

this is error is show in the function store
can help me 

thank yos so much for you❤️❤️❤️❤️❤️💕

_Originally posted by @mohammedALhakmy in https://github.com/spatie/laravel-permission/issues/408#issuecomment-1796168417_

**Comment by parallels999** [comment]
Seems like the same https://github.com/spatie/laravel-permission/issues/2530#issuecomment-1779319899

**Comment by tony3supriadi** [comment]
Your permissions data is array of string.
If your data is permission id please convert to array of numeric. But if your data is permission name using array of string it working.
